routes/util.go

```diff
@@ -3,15 +3,22 @@ package routes
 import (
 	"os"
 	"path/filepath"
+	"strings"
 )
 
+const defaultDescription = `Unnamed repository; edit this file 'description' to name the repository.`
+
 func getDescription(path string) (desc string) {
-	db, err := os.ReadFile(filepath.Join(path, "description"))
-	if err == nil {
-		desc = string(db)
-	} else {
-		desc = ""
+	data, err := os.ReadFile(filepath.Join(path, ".git", "description"))
```

Comment:
Hard-coding the ".git" suffix is incorrect as bare repos can very well exist without it (and that's how I have them setup, for what it's worth).

Reply:
Oh do you mean to say you normally point `legit` as bare repos? 🤔

Reply:
That's the intended use, yes. Clones will not work otherwise. I just realized the above highlighted line of code is looking for `foo/.git/description`; bare repos have it in `foo/description`. I suppose simply checking for both will "fix" the `getDescription` function.

Reply:
Yeah right now I'm just pointing my `legit` instance locally to `~/Projects` which are obviously not "bare" git repos 😅 -- Maybe we should support both? 🤔

Reply:
Apologies for the late reply, and a Happy New Year to you. :)

> Maybe we should support both?

Probably best, yes. The existing `getDescription` implementation can simply be augmented to first check for `foo/description`, if that's not present, check for `foo/.git/description`.

Reply:
Happy New Year to you too! 🥳

Reply:
I will however close this PR, as I've since forked your codebase (as previously discussed) and I decided to not implement this in my fork anyway and keep with the original design of serving up bare repos. 👌